src/map/lua/lua_baseentity.h

```diff
@@ -851,10 +851,12 @@ class CLuaBaseEntity
 
     bool actionQueueEmpty();
 
-    void castSpell(sol::object const& spell, sol::object const& entity); // forces a mob to cast a spell (parameter = spell ID, otherwise picks a spell from its list)
-    void useJobAbility(uint16 skillID, sol::object const& pet);          // forces a job ability use (players/pets only)
-    void useMobAbility(sol::variadic_args va);                           // forces a mob to use a mobability (parameter = skill ID)
-    bool hasTPMoves();
+    void  castSpell(sol::object const& spell, sol::object const& entity); // forces a mob to cast a spell (parameter = spell ID, otherwise picks a spell from its list)
+    void  useJobAbility(uint16 skillID, sol::object const& pet);          // forces a job ability use (players/pets only)
+    void  useMobAbility(sol::variadic_args va);                           // forces a mob to use a mobability (parameter = skill ID)
+    float getAbilityDistance(uint16 skillID);                             // Returns the specified distance for mob skill
```

Comment:
auto methodName() -> float;, so you don't have to realign everything

Reply:
Updated to
```lua
auto getAbilityDistance(uint16 skillID);
```

Reply:
Saw your reply (not sure why its disconnected from the feedback on the line)

Re-Updated to:
```cpp
    auto getAbilityDistance(uint16 skillID) -> float;                    // Returns the specified distance for mob skill
```